OpenLaTeXStudio/EditorModule/nbproject/project.xml

```diff
@@ -105,6 +105,14 @@
                     </run-dependency>
                 </dependency>
                 <dependency>
+                    <code-name-base>org.openide.util.ui</code-name-base>
+                    <build-prerequisite/>
+                    <compile-dependency/>
+                    <run-dependency>
+                        <specification-version>9.4.1</specification-version>
+                    </run-dependency>
+                </dependency>
+                <dependency>

```

Comment:
You're right. HelpCtx has been moved from org.openide.util to org.openide.util.ui and since the ui subpackage has not existed in 8.0.x, the build fails... I suppose it's OK for us to move to NB 8.1 since it has been out for some time now, I'll update the wikis and send info to mailing list.